app/views/todos/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<ul>
+  <% @incomplete_todos.each do |todo| %>
+    <li>
+      <span><%= link_to todo.title, todo_path(todo, todo: {completed: true}), method: :patch %></span>
+      <span><%= todo.due_date %></span>
+      <span><%= link_to "show", todo_path(todo) %></span>
+      <span><%= link_to "edit", edit_todo_path(todo) %></span>
+      <span><%= link_to "delete", todo_path(todo), method: :delete %></span>
+    </li>
+  <% end %>
+
+  <% @complete_todos.each do |todo| %>
+    <li class="completed">
```

Comment:
Nice use of a  CSS class to strikethrough the completed todos. It's simple and easy to follow what it does.